.aws/src/backfillLambda.ts

```diff
@@ -79,10 +80,62 @@ export class BackfillLambda extends Resource {
           },
         ],
         alarms: {
-          // TODO: set better alarm values
+          errors: {
+            // The backfill lambda will likely only run for 10-15 minutes total,
+            // so having more than one evaluation period is not necessary.
+            evaluationPeriods: 1,
+            comparisonOperator: 'GreaterThanOrEqualToThreshold',
+            period: 300, // 5 minutes
+            threshold: 50,
+            // TODO: remove this alarm from Dev when testing is complete
+            // actions: config.isDev
+            //   ? []
+            //   : [pagerDuty!.snsNonCriticalAlarmTopic.arn],
+
+            // For now, we need alerts in Dev, too
+            actions: [pagerDuty.snsNonCriticalAlarmTopic.arn],
+          },
+          invocations: {
+            evaluationPeriods: 1,
+            comparisonOperator: 'LessThanLowerOrGreaterThanUpperThreshold',
```

Comment:
we set this alarm to denote that lambda is getting invoked but not high enough to the extent of migration. 

would this alarm be a noise? for e.g lambda invoked for migrating one/two records, but not running actual migration,

(or) when the migration is at the end.

Reply:
we might also need 2 threshold, we have `LessThanLowerOrGreaterThanUpperThreshold` and threshold as 10,